apps/src/code-studio/showProjectAdmin.js

```diff
@@ -39,40 +39,22 @@ export default project => {
     $('#unpublished_warning').show();
   }
 
-  if ($('#feature_project').length && project.isProjectLevel()) {
-    $('#feature_project').click(function () {
-      var url = `/featured_projects/${project.getCurrentId()}/feature`;
+  if ($('#save_project').length && project.isProjectLevel()) {
+    $('#save_project').click(function () {
+      var url = `/featured_projects/${project.getCurrentId()}/save`;
       $.ajax({
         url: url,
         type: 'PUT',
         dataType: 'json',
         success: function (data) {
-          $('#unfeature_project').show();
-          $('#feature_project').hide();
+          $('#is_featured_project').show();
+          $('#save_project').hide();
         },
         error: function (data) {
           alert("Shucks. Something went wrong - this project wasn't featured.");
```

Comment:
Should this alert error message match the new functionality? e.g. "this project wasn't saved"

Reply:
Updated.